<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LegendChoice</class>
 <widget class="QDialog" name="LegendChoice">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>555</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>489</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>555</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Legend Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_8">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QFrame" name="frame_2">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>50</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget { background: white }</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_3" columnstretch="1,0">
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="horizontalSpacing">
         <number>9</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="m_datasourceTypeTitleLabel">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Legend Settings</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="m_imgLabel">
          <property name="minimumSize">
           <size>
            <width>62</width>
            <height>48</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>48</width>
            <height>48</height>
           </size>
          </property>
          <property name="text">
           <string>Imagem</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_14">
       <item>
        <widget class="QLabel" name="m_lblConnectedTo">
         <property name="text">
          <string>Connected To:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="m_cmbConnectedTo"/>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>178</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QWidget" name="m_widget" native="true">
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>180</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="m_grpSettings">
       <property name="title">
        <string>Settings</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <widget class="QLabel" name="m_lblSymbolSize">
              <property name="text">
               <string>Symbol size:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="m_txtSymbolSize">
              <property name="minimumSize">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>mm</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="m_layerTitleFont">
              <property name="text">
               <string>Layer Title Font:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QTextEdit" name="m_txtLayerFontTitle">
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>200</width>
                <height>20</height>
               </size>
              </property>
              <property name="frameShape">
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="verticalScrollBarPolicy">
               <enum>Qt::ScrollBarAlwaysOff</enum>
              </property>
              <property name="horizontalScrollBarPolicy">
               <enum>Qt::ScrollBarAlwaysOff</enum>
              </property>
              <property name="textInteractionFlags">
               <set>Qt::NoTextInteraction</set>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QPushButton" name="m_btnLayerTitleFont">
              <property name="minimumSize">
               <size>
                <width>31</width>
                <height>23</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>31</width>
                <height>23</height>
               </size>
              </property>
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="m_lblLayerTitleColor">
              <property name="text">
               <string>Layer Title Color:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLabel" name="m_lblLayerTitleColorFont">
              <property name="minimumSize">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QPushButton" name="m_btnLayerTitleColor">
              <property name="minimumSize">
               <size>
                <width>31</width>
                <height>23</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>31</width>
                <height>23</height>
               </size>
              </property>
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>13</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="0" column="0">
             <widget class="QLabel" name="m_lblRows">
              <property name="text">
               <string>Rows:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="m_txtRows">
              <property name="minimumSize">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>18</width>
                <height>17</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="m_lblHierarchyFont">
              <property name="text">
               <string>Hierarchy Font:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QTextEdit" name="m_txtHierarchyFont">
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>200</width>
                <height>20</height>
               </size>
              </property>
              <property name="frameShape">
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="verticalScrollBarPolicy">
               <enum>Qt::ScrollBarAlwaysOff</enum>
              </property>
              <property name="horizontalScrollBarPolicy">
               <enum>Qt::ScrollBarAlwaysOff</enum>
              </property>
              <property name="tabChangesFocus">
               <bool>false</bool>
              </property>
              <property name="textInteractionFlags">
               <set>Qt::NoTextInteraction</set>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QPushButton" name="m_btnHierarchyFont">
              <property name="minimumSize">
               <size>
                <width>31</width>
                <height>23</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>31</width>
                <height>23</height>
               </size>
              </property>
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="m_lblHierarchyColor">
              <property name="text">
               <string>Hierarchy Color:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLabel" name="m_lblHierarchyColorFont">
              <property name="minimumSize">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QPushButton" name="m_btnHierarchyFontColor">
              <property name="minimumSize">
               <size>
                <width>31</width>
                <height>23</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>31</width>
                <height>23</height>
               </size>
              </property>
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>13</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="m_grpDisplacement">
       <property name="title">
        <string>Displacement</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="m_lblDisplacementBorder">
              <property name="text">
               <string>Border:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="m_txtDisplacementBorder">
              <property name="minimumSize">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>mm</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="m_lblDisplacementBetweenSymbols">
              <property name="text">
               <string>Between Symbols:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="m_txtDisplacementBetweenSymbols">
              <property name="minimumSize">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>mm</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="m_lblDisplacementBetweenTitles">
              <property name="text">
               <string>Between Titles:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="m_txtDisplacementBetweenTitles">
              <property name="minimumSize">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>mm</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer_16">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>17</width>
              <height>17</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <widget class="QLabel" name="m_lblDisplacementBetweenColumns">
              <property name="text">
               <string>Between Columns:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="m_txtDisplacementBetweenColumns">
              <property name="minimumSize">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>mm</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="m_lblDisplacementGroupingPair">
              <property name="text">
               <string>Between Hierarchy Pair:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="m_txtDisplacementHierarchyPair">
              <property name="minimumSize">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>mm</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="m_lblDisplacementBetweenSymbolsAndTexts">
              <property name="text">
               <string>Between Symbols and Texts:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="m_txtDisplacementBetweenSymbolsAndTexts">
              <property name="minimumSize">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>mm</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer_13">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>13</width>
              <height>17</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="helpPushButton">
         <property name="text">
          <string>&amp;Help</string>
         </property>
         <property name="shortcut">
          <string>Alt+H</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>24</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="m_okPushButton">
         <property name="text">
          <string>OK</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_cancelPushButton">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
